src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputJavadocMethodAndMissingJavadocMethod.java

```diff
@@ -0,0 +1,221 @@
+package com.google.checkstyle.test.chapter7javadoc.rule73wherejavadocrequired;
+
+/**
+ * The following is a bad tag.
+ * @mytag Hello
+ */
+public class InputJavadocMethodAndMissingJavadocMethod extends OverrideClass
+{
```

Comment:
Used only single input for `JavadocMethod` & `MissingJavadocMethod` modules, as they had the same input file before. They also work almost in the same way. They both logs violations for missing javadoc, I dont why we have two different modules for it